setup.py

```diff
@@ -460,17 +460,22 @@ def check_LAPACK(self):
     def cython_extensionlists(self):
         extra_link_flags = self.static_link_gcc + self.flag_32bit
         
-        #Cythonize main modules
-        ext_list = cythonize(["assimulo"+os.path.sep+"*.pyx"], 
-                             include_path=[".","assimulo"])
-
-        #Cythonize Solvers
+        # Cythonize main modules 
+        ext_list = cythonize([os.path.join("assimulo", "explicit_ode.pyx")], 
+                             include_path=[".", "assimulo"], force = True)
+        ext_list[-1].include_dirs += ["assimulo", self.incdirs]
+        ext_list[-1].sources += [os.path.join("assimulo", "ode_event_locator.c")]
+        ext_list[-1].name = "assimulo.explicit_ode"
```

Comment:
Do you know why we need a name here, while not for the others?

Reply:
Actually, after testing it again, it appears redundant. Removed it.